src/os/bluestore/BlueFS.cc

```diff
@@ -741,8 +742,6 @@ int BlueFS::_read(
     }
 
     dout(30) << __func__ << " result chunk (" << r << " bytes):\n";
-    bufferlist t;
-    t.substr_of(buf->bl, off - buf->bl_off, r);
```

Comment:
Your patch might have substring taken from the different offset comparing to the original code